duckscript_sdk/src/sdk/std/string/camel_case/mod.rs

```diff
@@ -0,0 +1,44 @@
+use crate::utils::pckg;
+use duckscript::types::command::{Command, CommandResult};
+
+#[cfg(test)]
+#[path = "./mod_test.rs"]
+mod mod_test;
+
+#[derive(Clone)]
+pub(crate) struct CommandImpl {
+    package: String,
+}
+
+impl Command for CommandImpl {
+    fn name(&self) -> String {
+        pckg::concat(&self.package, "CamelCase")
+    }
+
+    fn aliases(&self) -> Vec<String> {
+        vec!["camel_case".to_string()]
+    }
+
+    fn help(&self) -> String {
+        include_str!("help.md").to_string()
+    }
+
+    fn clone_and_box(&self) -> Box<dyn Command> {
+        Box::new((*self).clone())
+    }
+
+    fn run(&self, arguments: Vec<String>) -> CommandResult {
+        if arguments.is_empty() {
+            CommandResult::Error("No arguments provided.".to_string())
+        } else {
+            let snake_case = heck::CamelCase::to_camel_case(arguments[0].as_str());
```

Comment:
you mean camel_case, not snake_case?
to avoid such copy paste bugs, i usually go with value/output/... stuff like that :)

Reply:
renamed all similar vars to "value"